### PR TITLE
Multiple_Streamtube_Darrieus_Turbines function

### PR DESCRIPTION
**Pull request by CamillaScotto**
turbine_Darrieus_flusso_multiplo function created.

### CODE REVIEW

**Comment by CamillaScotto** [comment]
This function implements the multiple streamtubes theory which gives an accurate prediction of the wind velocity variations across the Darrieus rotor.

INPUT :
                 
-  (alphamax) = angle of attack at the stall of the considered airfoil               
-  (c) = chord of the blade                                                    
-  (R) = blade radius                                                          
-  (Cla) = slope coefficient of the Cl curve for the considered airfoil        
-  (N) = number of blades                                                      
-  (Cd) = drag coefficient of the considered airfoil        

OUTPUT :
                  
- (cp) = vector of power coefficient values for the turbine                                                                      
- (cq) = vector of torque coefficient values for the turbine                                                                      
- (lambdav) = vector of tip speed values corresponding to the cp and cq coefficients

**Comment by EttoreSaetta** [comment]
The name of the function should be in English, the same for the name of the documentation file.

File: Multiple_Streamtube_Darrieus_Turbines_Theory.m [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+%Allocating vector of max values
+max_v = zeros(numel(lambdav),1);
+alphadeg = rad2deg(alpha); % [deg]
+
+%Find max alpha for each row of the matrix alpha
+for h = 1 : numel(phiv)
+    max_v(h) = max(alphadeg(h,:));
+end
+
+contatorelambdamin = 0; %intial value for lambda min index
```
Comment (EttoreSaetta) [inline Multiple_Streamtube_Darrieus_Turbines_Theory.m]: Just change this one to `counterlambdamin`.

**Review by EttoreSaetta — APPROVED** [review]
LGTM.
Just one correction, then you are allowed to merge this branch.

**Pull request merged**